docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -434,35 +434,35 @@ where t is the period for which transition the is estimated (a year) eg. 365 day
     - Distribution
     - Note
     - Source
-  * - :math:`duration_\text{treated SAM}`
-    - 0-6 months old
+  * - :math:`\text{time to recovery}_\text{effectively treated SAM}`
+    - 0-6 months olds
```

Comment:
```suggestion
    - 0-6 months old
```
The s at the end is inconsistent with the other table entries

Reply:
Thanks! Updating.